etcdctl/ctlv3/command/ep_command.go

```diff
@@ -44,6 +48,9 @@ func newEpHealthCommand() *cobra.Command {
 		Short: "Checks the healthiness of endpoints specified in `--endpoints` flag",
 		Run:   epHealthCommandFunc,
 	}
+
+	cmd.Flags().StringVar(&healthCheckKey, "check-key", "health", "A key used as a parameter of get request")
```

Comment:
health-key? or health-check-key?

A key that is accessed by the health check command. Make sure the command have at least read access to the key. The default key is "health".

/cc @heyitsanthony better words?